src/kirby/scss/web/_ionic.scss

```diff
@@ -1,39 +1,60 @@
-@import "../themes/colors";
-@import "../base/functions";
+@import '../themes/colors';
```

Comment:
Please revert the formatting/line indentation back to original, or else it will clutter up the version history...